---
bool flag
baxterplaysyahtzee/ObjectInfo objInfo


